setup-e2e-tests.ps1

```diff
@@ -25,11 +25,11 @@ Remove-Item -Recurse -Force ./Azure.Functions.Cli -ErrorAction Ignore
 
 if (-not (Test-Path env:CORE_TOOLS_URL)) 
 { 
-  $env:CORE_TOOLS_URL = "https://functionsclibuilds.blob.core.windows.net/builds/2/latest/Azure.Functions.Cli.win-x86.zip"
+  $env:CORE_TOOLS_URL = "https://functionsclibuilds.blob.core.windows.net/builds/3/latest/Azure.Functions.Cli.win-x86.zip"
```

Comment:
Might be worth it to keep runs on V2 as well. Can we have test runts for V1 and V2 for now?

Reply:
This is just for the "v2.x" branch of the Node.js worker (which only runs with V3 bits) - so running against V2 was not a valid scenario!

Reply:
gotcha!